src/Symfony/Component/HttpKernel/KernelInterface.php

```diff
@@ -18,8 +18,6 @@
 /**
  * The Kernel is the heart of the Symfony system.
  *
- * It manages an environment made of bundles.
```

Comment:
You are right. I've rebased this onto 4.0, these changes make more sense there. And I have changed the wording of this sentence instead of deleting it.